src/Compiler/Compiler.php

```diff
@@ -262,6 +262,7 @@ private static function getSignatures()
                 )
             ),
             'sort' => $anythingToList,
+            'sortDesc' => $anythingToList,
```

Comment:
My plan was to use "rsort" for the reverse sort, based on the PHP function:
http://php.net/manual/en/function.rsort.php

My thinking was that developers at Datto will be familiar with PHP and JavaScript, so I've been trying to adopt the same naming conventions... along with some other tiny details, like following the same operator-precedence.